api/graphite.go

```diff
@@ -1200,3 +1200,56 @@ func (s *Server) graphiteTagDelSeries(ctx *middleware.Context, request models.Gr
 
 	response.Write(ctx, response.NewJson(200, res, ""))
 }
+
+// showPlan attempts to create a Plan given a /render target query.
+// If the Plan creation is succesful it returns 200, JSON marshaling of Plan.
+// Otherwise, it returns 400, error details.
+// This is needed to determine if a query cannot be resolved in localOnly mode.
+func (s *Server) showPlan(ctx *middleware.Context, request models.GraphiteRender) {
+    // note: the model is already validated to assure at least one of them has len >0
```

Comment:
This is true for this endpoint too, right?

Reply:
I hope so.  The route definition is basically the same as the `/render` endpoint.